docs/user/technical-reference/05-10-application-connector-chart.md

```diff
@@ -0,0 +1,13 @@
+# Application Connector chart
+
+To configure the Application Connector (AC) chart, override the default values of its [`values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/application-connector/values.yaml) file. This document describes parameters that you can configure.
+
+>**TIP:** See how to [change Kyma settings](https://github.com/kyma-project/kyma/blob/main/docs/04-operation-guides/operations/03-change-kyma-config-values.md).
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.disableLegacyConnectivity** | Disables the default standalone (legacy) connectivity components and enables the Compass mode. | `false` |
```

Comment:
We will need to update this list as soon the CR schema is defined. DisableLegacyConnectivity flag will be removed